src/cata_tiles.cpp

```diff
@@ -270,6 +270,11 @@ void cata_tiles::load_tilejson_from_file(std::ifstream &f, const std::string &im
         JsonObject curr_info = info.next_object();
         tile_height = curr_info.get_int("height");
         tile_width = curr_info.get_int("width");
+        if (curr_info.has_bool("iso")) {
```

Comment:
I saw a JsonObject.get_something() cause the whole json loading process to fail when something was missing. We should probably make use of those defaults in more places...